rita_client/src/rita_loop/mod.rs

```diff
@@ -115,7 +115,7 @@ impl Handler<Tick> for RitaLoop {
         trace!("Client Tick!");
 
         if metrics_permitted() {
-            send_udp_heartbeat();
+            send_heartbeat_loop();
```

Comment:
This is going to spawn a new thread with every iteration of the loop, put it next to where we call start_rita_loop (use the new rita loop below as an example).